### PR TITLE
Add Stronghold to Security Section

### LINKED ISSUE #199: Add Stronghold to Security section
### I'm Submitting a ...

```
[ ] Bug report
[x] Tool/language/etc documentation request
```
### Tool/Language/etc

[stronghold](https://github.com/alichtman/stronghold) -- The easiest way to securely configure MacOS.

### PR DESCRIPTION
**Pull request by alichtman**
Fixes #199

### CODE REVIEW

File: Security/README.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 
+* secure system settings
 * a virus scanner
 * a firewall
 * disk encryption
 
-## Virus scanner
+## Secure System Settings
+
+Check out [stronghold](https://www.github.com/alichtman/stronghold) for the easiest wayt to configure MacOS security settings from the terminal.
```
Comment (hugovk) [inline Security/README.md]: stronghold -> Stronghold
wayt -> way
MacOS -> macOS

Comment (alichtman) [reply Security/README.md]: Oops, my bad on that extra 't'

Fixing everything else.

Comment (alichtman) [reply Security/README.md]: Done.

**Comment by Kyslik** [comment]
LGTM

**Comment by sb2nov** [comment]
LGTM

**Pull request merged**